### PR TITLE
Increases audittrail-adapter memory

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
Signed-off-by: Alexander Yastrebov <alexander.yastrebov@zalando.de>

### CODE REVIEW

File: cluster/manifests/audittrail-adapter/daemonset.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ metadata:
   namespace: kube-system
   labels:
     application: audittrail-adapter
-    version: master-24
+    version: master-26
```
Comment (AlexanderYastrebov) [inline cluster/manifests/audittrail-adapter/daemonset.yaml:9]: version label was not updated on last image version update

**Comment by szuecs** [comment]
:+1:

File: cluster/manifests/audittrail-adapter/daemonset.yaml [hunk 2]
```diff
@@ -36,7 +36,7 @@ spec:
       hostNetwork: true
       containers:
       - name: audittrail-adapter
-        image: pierone.stups.zalan.do/teapot/audittrail-adapter:master-25
+        image: pierone.stups.zalan.do/teapot/audittrail-adapter:master-26
```
Comment (mikkeloscar) [inline cluster/manifests/audittrail-adapter/daemonset.yaml:39]: What changed in the image?

Comment (AlexanderYastrebov) [reply cluster/manifests/audittrail-adapter/daemonset.yaml:39]: `[AUTOMATED] Update Alpine base image 3.12 -> 3.13`

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**